docs/developers/tutorial.md

```diff
@@ -654,6 +654,7 @@ The expansion rules for `github:org/repo` are the following:
 | `github:org/repo`                      | `scalafix/rules/src/main/scala/fix/Repo.scala`                           |
 | `github:org/some-repo`                 | `scalafix/rules/src/main/scala/fix/SomeRepo.scala`                       |
 | `github:org/repo/RuleName`             | `scalafix/rules/src/main/scala/fix/RuleName.scala`                       |
+| `github:org/repo/package.RuleName`     | `scalafix/rules/src/main/scala/fix/package/RuleName.scala`               |
```

Comment:
if package is explicit, shouldn't we remove the `fix` prefix?
```suggestion
| `github:org/repo/package.RuleName`     | `scalafix/rules/src/main/scala/package/RuleName.scala`               |
```

Reply:
Looks that scalafix requires the root package to be `fix`.
When only the rule name is provided, it searches in the `fix` package, this changes keep the same behavior

Reply:
> Looks that scalafix requires the root package to be fix.

I am not aware of this constraint in scalafix-core, and none of the built-in rules is rooted in `fix`. Can you elaborate where you observed this limitation?

>  When only the rule name is provided, it searches in the fix package, this changes keep the same behavior

I agree that the existing behavior should remain, but it feels strange that the user provides a package that is only a suffix. If the example would look like this, would it make more sense?
```suggestion
| `github:org/repo/fix.package.RuleName`     | `scalafix/rules/src/main/scala/fix/package/RuleName.scala`               |
```

Reply:
My bad, I double checked `scalafix` uses the rule name only, regardless of the class name/package.
I agree with the suggestion. Will update the PR